apps/desktop/src/renderer/styles/globals.css

```
@@ -739,6 +739,18 @@ th {
   from { opacity: 0; transform: translate(-50%, -8px) scale(0.96); }
   to   { opacity: 1; transform: translate(-50%, 0) scale(1); }
 }
+
+/* Turn status pulse (issue #646: 等待你的确认) */
+@keyframes turnPulse {
```

Stylelint (17.14.0)

[error] Expected keyframe name "turnPulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)